app/presenters/hyrax/pcdm_member_presenter_factory.rb

```diff
@@ -97,7 +97,7 @@ def work_presenters
     # @return
     def presenter_for(document:, ability:)
       case document['has_model_ssim'].first
-      when Hyrax::FileSet.name, ::FileSet.name # ActiveFedora FileSet within a Valkyrie Resource
+      when Hyrax::FileSet.name, '::FileSet'.safe_constantize&.name # ActiveFedora FileSet within a Valkyrie Resource
```

Comment:
```
require 'pathname'
value = nil
case value
when Pathname, nil
  puts "WHEN"
else
  puts "Else"
end
```

The above will output `WHEN` meaning that the safe constantize will begin to return a `Hyrax::FileSetPresenter.new` if there is no `document['has_model_ssim'].first`.

I'm not certain if that is an issue but it's something I just learn.